frontend/views/Browser.vue

```diff
@@ -144,11 +144,17 @@
               </b-dropdown>
             </b-table-column>
           </template>
+        </b-table>
 
-          <template slot="bottom-left">
+        <section class="is-flex is-justify-between">
+          <div>
             <span>{{ lang('Selected', checked.length, totalCount) }}</span>
-          </template>
-        </b-table>
+          </div>
+          <div v-if="(showAllEntries || hasFilteredEntries) ">
+            <input type="checkbox" id="checkbox" v-model="showAllEntries" @click="loadFiles">
+            <label for="checkbox"> {{ lang('show all files/folders') }}</label>
```

Comment:
Translations string should be something simpler, maybe 'show hidden'. 

TODO: add this to all translation files